ipywidgets_bokeh/package.json

```diff
@@ -14,15 +14,15 @@
     "url": "https://github.com/bokeh/ipywidgets_bokeh.git"
   },
   "dependencies": {
-    "@bokeh/bokehjs": "~2.0.0",
+    "@bokeh/bokehjs": "^2.4.0",
```

Comment:
Is this change intentional? Given a version `x.y.z`, `^` operator allows changes at `y`, whereas `~` operator allows changes at `z`. Given changes to `install_requires`, I presume `~` should be kept.

Reply:
You're right, thanks!